src/devices/ServoMotor/samples/README.md

```diff
@@ -0,0 +1,20 @@
+# Servo Motor Sample
+Connect the servo motor to the dev board with ground, power and control signal.  Be aware servo motors consume lots of current. You may need to power the servo motor to an external power source to prevent damage to the dev board.
+
+**NOTE**: The following image shows the servo is connected to a regular GPIO and not a hardware PWM pin.
+
+![schema](./servomotor.png)
+
+## Sample Code
+
+The sample code provides helpful examples where you can control the servo motor's position by pulse width or angle.
+
+### Writing Pulse Width
+This example allows you to manually view the full left and right positions based on the pulse width signal entered.  Many times, servo motors do not match the exact range based on datasheets and/or have to be calibrated using the screw on side of servo motor.
```

Comment:
this should be in the comment above WritePulseWidth (specifically that this method is used for calibration)

Reply:
> this should be in the comment above WritePulseWidth

Do you mean add to remarks of method summary?  That would probably be a good idea.

I don't want to give the impression that `WritePulseWidth` method should only be used for calibration as some users might base their apps on pulse width values instead of angles (in degrees).  Having the two methods offers them options.

Reply:
@shaggygi what would be the usage for using pulse width? I don't see that there is any use other than calibration. Worst case scenarios users can use PWM directly if they for whatever reason need to have more control